Qiskit open-source index page (#1725)
Qiskit/documentation: docs/open-source/index.mdx

---
title: Open source

> description: Qiskit open-source introduction

Reviewer: Needs to be >= 50 and <= 160 chars. Abby M and Abby C probably have better suggestions, but maybe:

```suggestion
description: Learn how to get involved with the Qiskit open-source community to shape the future of quantum computing.
```

Reviewer 2: I'm good with this but @javabster is the true expert :-)

---

# Open source

> Since 2016, IBM Quantum has harnessed the power of community-driven innovation by making quantum computing resources accessible to everyone via the cloud.

Reviewer 2: ```suggestion
Since 2016, IBM Quantum&trade; has harnessed the power of community-driven innovation by making quantum computing resources accessible to everyone via the cloud.
```

This groundbreaking initiative includes open-sourcing key parts of the quantum software stack and introducing Qiskit, a versatile quantum computing framework.

> Today, Qiskit is the most popular quantum framework, thanks to its vibrant community.

Reviewer 3: this sentence implies that the success is only due to the vibrant community, could we say "thanks in part to its vibrant community" or "thanks to its vibrant community and xyz other things"

The Qiskit community is a dynamic and welcoming group of developers, scientists, and enthusiasts collaborating to advance software, research, education, and business through quantum computing.

> By making quantum computing open and accessible, we accelerate innovation and discover new applications for this transformative technology.

Reviewer 2: Trying to avoid too many repetitions of "quantum computing" in a row...

```suggestion
By making this transformative technology open and accessible, we accelerate innovation and discover new applications.
```

> As quantum technology evolves, contributions from the open-source Qiskit Ecosystem projects and developers are crucial in shaping its future.

Author: Link to Qiskit Ecosystem site?

Reviewer: We're no longer allowed to capitalize the E hah

```suggestion
As quantum technology evolves, contributions from the open-source [Qiskit ecosystem](https://www.ibm.com/quantum/ecosystem) projects and developers are crucial in shaping its future.
```

Reviewer 3: the grammar sounds a bit off here: "contributions from...". the word "from" feels off. Maybe something like "as quantum technology evolves, contributions to the Qiskit SDK and the open-source Qiskit ecosystem are crucial..."

> The development of a dense ecosystem is vital not only for extending Qiskit's capabilities but also for spreading involvement and engagement across various sectors.

Reviewer 3: I think this sentence about the various sectors an be dropped, it feels a bit vague and i think the other sentences put the main point across better

> By joining the Qiskit community, you can participate in the development of quantum technologies.

Reviewer: ```suggestion

By joining the Qiskit community, you can participate in the development of quantum technologies.
```

> Whether you are a seasoned quantum programmer or just beginning your quantum journey, Qiskit provides the support and inspiration needed to push boundaries.

Reviewer 2: Alas - IBM Style asks us not to use "just" bc of its trickiness in translation; we can either remove it, or replace with "only" if that sounds better 
```suggestion
Whether you are a seasoned quantum programmer or beginning your quantum journey, Qiskit provides the support and inspiration needed to push boundaries.
```

> We welcome all kinds of contributions, from code in PRs to community support on [Slack](https://docs.quantum.ibm.com/support#qiskit), ensuring everyone can play a role in advancing quantum technology.

Reviewer 2: ```suggestion
We welcome all kinds of contributions, from code in pull requests to community support on [Slack](https://docs.quantum.ibm.com/support#qiskit), ensuring everyone can play a role in advancing quantum technology.
```

> We welcome all kinds of contributions, from code in pull requests to community support on [Slack](https://docs.quantum.ibm.com/support#qiskit), ensuring everyone can play a role in advancing quantum technology.

Reviewer 2: @Eric-Arellano can/should we use an internal link here?

```suggestion
We welcome all kinds of contributions, from code in pull requests to community support on [Slack](/support#qiskit), ensuring everyone can play a role in advancing quantum technology.
```

Reviewer: Yes, thanks.

> We welcome all kinds of contributions, from code in PRs to community support on Slack, ensuring everyone can play a role in advancing quantum technology.

Reviewer: ```suggestion
We welcome all kinds of contributions, from code in PRs to community support on [Slack](https://docs.quantum.ibm.com/support#qiskit), ensuring everyone can play a role in advancing quantum technology.
```

Reviewer 3: should we say "pull requests" instead of PRs? in case people are unfamiliar with the acronym

Author: That was a bit my point, "from expert to not". _PR_ is talking to the expert. But maybe it is trying too hard to look too clever.

Reviewer: > PR is talking to the expert.

No need to gatekeep. Even if beginners won't be contributing PRs, it's good for them to be able to understand the document. For example, maybe it helps to learn what PRs are so they in several months start contributing.

---
title: Open source
description: Qiskit open-source introduction
---

# Open source

Since 2016, IBM Quantum&trade; has harnessed the power of community-driven innovation by making quantum computing resources accessible to everyone via the cloud.
This groundbreaking initiative includes open-sourcing key parts of the quantum software stack and introducing Qiskit, a versatile quantum computing framework.

> Today, Qiskit is the most popular quantum framework, thanks to its vibrant community.

Reviewer 2: ```suggestion
Today, Qiskit is the most popular quantum framework, thanks in part to its vibrant community.
```

The Qiskit community is a dynamic and welcoming group of developers, scientists, and enthusiasts collaborating to advance software, research, education, and business through quantum computing.
By making this transformative technology open and accessible, we accelerate innovation and discover new applications.

As quantum technology evolves, contributions from the open-source [Qiskit ecosystem](https://www.ibm.com/quantum/ecosystem) projects and developers are crucial in shaping its future.

> The development of a dense ecosystem is vital not only for extending Qiskit's capabilities but also for spreading involvement and engagement across various sectors.

Reviewer 2: ```suggestion
```
